digitalocean/participate.yaml

```diff
@@ -0,0 +1,32 @@
+# https://www.digitalocean.com/docs/app-platform/references/app-specification-reference/
+domains:
+- domain: staging.participate.whatwg.org
+  type: PRIMARY
+  zone: whatwg.org
+name: participate
+region: nyc
+services:
+- dockerfile_path: Dockerfile
+  envs:
+  - key: PORT
+    scope: RUN_TIME
+    value: '8080'
```

Comment:
Didn't we decide to stick with 3000? Or should we remove the `ENV=3000` in the Dockerfile?

Reply:
Yeah, it's a bit confused now. When I added `ENV PORT=3000` to the Dockerfile I thought that would allow an environment variable `PORT` set in the environment invoking `docker run` to control which port is used and exposed. But I don't think that's actually how it works. I think this can be simplified, but I need to experiment until I understand it.

Reply:
Alright, while the `ENV PORT=3000` and `EXPORT PORT` isn't necessary, it does mean that `docker run` locally works without manually mapping the port. Otherwise one has to add `-p 3000:3000/tcp` to make it work. So let's keep that.

But this here can be simplified. Setting `http_port` implicitly adds a `PORT` environment variable, so just `http_port: 3000` works. Even removing that and defaulting to 8080 should work, but then it's all implicit and it will be harder to figure out why it even works.